cetmix_tower_server/models/cx_tower_command.py

```diff
@@ -86,7 +87,11 @@ def _selection_action(self):
         string="OSes",
     )
     note = fields.Text()
-
+    secret_ids = fields.Many2many(
+        comodel_name="cx.tower.key",
+        relation="cx_tower_secret_command_rel",
+        column1="command_id",
+    )
```

Comment:
we are overriding the field from the secret.mixin model and declared these parameters in that model, so there is no need to specify this because when overriding the field only the necessary changes